db/migrate/20180724133041_create_amusement_park.rb

```diff
@@ -0,0 +1,18 @@
+class CreateAmusementPark < ActiveRecord::Migration[5.2]
+  def change
+    create_table :amusement_parks do |t|
+      t.string :name, null: false
+       t.string :address, null: false
```

Comment:
there's an extra space here and for the lines below it. this should be indented to lined up with the line above it